definitions/k8s-resource/labels.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: builtin
+kind: LabelTransformer
+metadata:
+  name: labelTransformer
+labels:
+  app.kubernetes.io/component: kyverno
+  app.kubernetes.io/instance: kyverno
+  app.kubernetes.io/managed-by: Kustomize
+  app.kubernetes.io/name: kyverno
+  app.kubernetes.io/part-of: kyverno
+  app.kubernetes.io/version: v1.3.6-rc1
```

Comment:
If a new version is out, does it mean we need to update this label in multiple places?
`definitions/k8s-resource/labels.yaml`
`definitions/manifest/labels.yaml`
`definitions/kustomization.yaml`

I can see these are the files that need to be updated, is there a way to manage them centrally?

Reply:
I put the label transformer into the main kustomization file.